### PR TITLE
YAML name enhancements for JavaToolInstaller and related tasks

### PR DESCRIPTION
**Pull request by DavidStaheli**

### CODE REVIEW

File: Tasks/JavaToolInstaller/task.json [hunk 1]
```diff
@@ -11,116 +11,116 @@
...
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 0,
-        "Minor": 126,
-        "Patch": 1
+        "Major": 1,
+        "Minor": 127,
+        "Patch": 0
     },
     "preview": true,
```
Comment (madhurig) [inline Tasks/JavaToolInstaller/task.json]: If we are setting major version to 1, it makes sense to remove the "preview" flag.

**Comment by stephenmichaelf** [comment]
@DavidStaheli Is this going in or can we close it?

**Comment by DavidStaheli** [comment]
Ah, I forgot about this one.  Do you mind if we keep it around for S138 and I'll get to it?

**Comment by stephenmichaelf** [comment]
Sounds good thanks! Was just going through all the tasks PRs so we can close ones that aren't needed any more :)

**Comment by joshmgross** [comment]
@DavidStaheli Is this still going on?

**Pull request closed without merging**